Mathlib/Topology/Algebra/IsUniformGroup/Basic.lean

```diff
@@ -78,6 +78,30 @@ end IsUniformGroup
 
 end Cauchy
 
+namespace IsRightUniformGroup
+
+variable {G : Type*} [Group G] [UniformSpace G] [IsRightUniformGroup G]
+
+open scoped RightActions in
```

Comment:
Let's drop this here and instead add it as the first line of the proof (since it is needed only for the proof and not the statement)
```suggestion
```